lib/get_user_move.ex

```diff
@@ -36,7 +36,7 @@ defmodule GetUserMove do
       test_element
 
     else
```

Comment:
Maybe move this input_checker function to its own module since two things depend on it?

Reply:
Also, if the underscore means that it's a private function you probably want to remove it now

Reply:
I'm a little confused about the use of the underscores throughout. A leading underscore for a private function is a Python convention right? In Elixir though, you use the `defp` keyword to define a private function. Also, leading underscores tell Elixir not to import the function by default. So for instance when you say `import GetUserMove` you're importing everything from the `GetUserMove` module that doesn't start with an underscore, which in this case is just `get_move/2`. But then on lines 22 and 23, you have to explicitly go and grab those functions that weren't imported – why not just import them from the beginning?

Reply:
IoFunctions was created to handle input checkers, sluggers, and excluders

Reply:
_scores will be used for functions not meant to be exported to other modules. An important note is that these are not private functions but simply not imported by default, making them testable. The _score convention was used in place of private functions in case said functions would want to be used outside of their module later.